actions/lib/actions.py

```diff
@@ -12,4 +12,9 @@ def _get_client(self):
         username = self.config['username']
         password = self.config['password']
 
-        return sn.Client(instance=instance_name, user=username, password=password)
+        s = sn.Client(instance=instance_name, user=username, password=password)
```

Comment:
I believe a more explicit name would be better here. Perhaps `client` or something similar.